bitnami/thanos/templates/bucketweb/deployment.yaml

```
@@ -76,6 +76,10 @@ spec:
             {{- if .Values.bucketweb.extraFlags }}
             {{- .Values.bucketweb.extraFlags | toYaml | nindent 12 }}
             {{- end }}
+          {{- if .Values.bucketweb.extraEnv }}
+          env:
+            {{- toYaml .Values.bucketweb.extraEnv | nindent 12 }}
```

Something I just noticed is that we're using `nindent` here. The difference between `nindent` and `indent` is that it begins with a newline, so the rendered output would look like this:

```yaml
env:
  
  - name: env1
    value: test
```

Could you fix this and other lines where `nindent` is used? I.e.:

```suggestion
          env: {{- toYaml .Values.bucketweb.extraEnv | nindent 12 }}
```